src/components/InputErrorMessage/index.jsx

```diff
@@ -2,11 +2,20 @@ import React from "react";
 import "./index.scss";
 import patchExclamationFill from "../../images/icons/patchExclamationFill.svg";
 
-export default function InputErrorMessage({ message }) {
-  return (
-    <div className="errorMessage" role="alert">
-      <img src={patchExclamationFill} alt="Error Logo" width="32" height="32" />
-      <p>{message}</p>
-    </div>
-  );
+export default function InputErrorMessage({ errors, name }) {
+  if (errors && errors[name]) {
```

Comment:
avoid nesting

if (!errors || !errors[name]) {
return null;
}
return x

Reply:
okay, but I need `&&` as there might not be an error with that specific name provided, and in that case, `null` should be returned.